mmv1/third_party/terraform/website/docs/guides/using_write_only_attributes.html.markdown

```diff
@@ -0,0 +1,156 @@
+---
+page_title: "Use write-only attributes in the Google Cloud provider"
+description: |-
+  How to use write-only attributes in the Google Cloud provider
+---
+
+# Write-only attributes in the Google Cloud provider
+
+Write-only attributes allow users to access and use data in their configurations without that data being stored in Terraform state.
+
+Write-only attributes are available in Terraform v1.11 and later. For more information, see the [official HashiCorp documentation for Write-only Attributes](https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/write-only-arguments).
+
+To mark the launch of write-only attributes, the Google Cloud provider has added the following write-only attributes:
+- [`google_compute_disk: disk_encryption_key.raw_key_wo`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_disk.html#raw_key_wo)
+- [`google_compute_disk: disk_encryption_key.rsa_encrypted_key_wo`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_disk.html#rsa_encryption_key_wo)
+- [`google_compute_region_disk: disk_encryption_key.raw_key_wo`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_region_disk.html#raw_key_wo)
+- [`google_sql_user: password_wo`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_user#password-1)
+- [`google_secret_manager_secret_version: secret_data_wo`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/secret_manager_secret_version#secret_data_wo)
+- [`google_bigquery_data_transfer_config: sensitive_params.secret_access_key_wo`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_data_transfer_config#secret_access_key_wo)
+
+These were chosen due to them being marked as sensitive already in the provider. Although sensitive attributes do not appear in `terraform plan`, they are still stored in the Terraform state. Write-only attributes allow users to access and use data in their configurations without that data being stored in Terraform state.
+
+## Use the Google Cloud provider's new write-only attributes
+
+The following sections show how to use the new write-only attributes in the Google Cloud provider.
+
+### Applying a write-only attribute
+
+The following example shows how to apply a write-only attribute. All write-only attributes are marked with the `wo` suffix and can not be used with the `sensitive` attribute.
```

Comment:
This should probably be more clear- what sensitive attribute?

Reply:
recent commit made it more clear the different between write-only and non writeonly